src/test/java/teammates/test/cases/storage/FeedbackQuestionsDbTest.java

```diff
@@ -23,6 +23,18 @@
 
 public class FeedbackQuestionsDbTest extends BaseComponentTestCase {
     private static final FeedbackQuestionsDb fqDb = new FeedbackQuestionsDb();
+    
+    private class FeedbackQuestionAttributesExt extends FeedbackQuestionAttributes {
```

Comment:
Name this as `FeedbackQuestionAttributesWithModifiableTimestamps` / `ModifiableTimestampsFeedbackQuestionAttributes`? 

Can't really think of a good name for this too.